src/utils/shell.ts

```diff
@@ -62,16 +59,13 @@ export const runCommand = (command, options = {}) => {
 
   // This ensures that an active spinner stays at the bottom of the commandline
   // even though the actual framework command might be outputting stuff
-  // @ts-expect-error TS(7006) FIXME: Parameter 'writeStream' implicitly has an 'any' ty... Remove this comment to see the full error message
-  const pipeDataWithSpinner = (writeStream, chunk) => {
-    if (spinner && spinner.isSpinning) {
+  const pipeDataWithSpinner = (writeStream: NodeJS.WriteStream, chunk: any) => {
+    if (spinner?.isSpinning) {
       spinner.clear()
-      spinner.isSilent = true
     }
     writeStream.write(chunk, () => {
-      if (spinner && spinner.isSpinning) {
-        spinner.isSilent = false
-        spinner.render()
+      if (spinner?.isSpinning) {
+        spinner.start()
       }
     })
   }
```

Comment:
Feel free to compare the docs for ora and yocto-spinner and double-check me here, but I think this should reproduce the behaviour. Of course I'll poke at it manually before considering merging.